tests/pr_test.go

```diff
@@ -48,6 +48,13 @@ func TestRunCompleteExample(t *testing.T) {
 	t.Parallel()
 
 	options := setupOptions(t, "kms-key")
+
+	options.TerraformVars = map[string]interface{}{
+		"existing_secrets_manager_crn": permanentResources["secretsManagerCRN"],
+		"prefix":                       options.Prefix,
+		"region":                       options.Region,
```

Comment:
`region` here will have to be hard coded to the region our permanent SM instance is in